pkg/notifications/email.go

```diff
@@ -62,7 +64,14 @@ func newEmailNotifier(c *cobra.Command, acceptedLogLevels []log.Level) t.Notifie
 }
 
 func (e *emailTypeNotifier) buildMessage(entries []*log.Entry) []byte {
-	emailSubject := "Watchtower updates"
+	// If variable SubjectTag is empty define emailSubject as "Watchtower updates".
+	// If variable SubjectTag is set define emailSubject as "$SubjectTag Watchtower updates"
+	// For example: SubjectTag=[Server Munich] -> "[Server Munich] Watchtower updates ..."
+	if SubjectTag == nil {
```

Comment:
a string is never nil, however it could be "" if left unset. however, if you go for the suggestion with replacing the whole subject tag, no comparison is needed at all. Also, I'd prefer if you dont leave comments in the code explaining something that is perfectly understandable from reading the code itself 👍  😄

Reply:
Ok. Thx. I will change it or can you change it